components/sup/src/manager/service/mod.rs

```diff
@@ -685,6 +685,21 @@ impl Service {
         false
     }
 
+    pub fn suitability(&mut self) -> Option<u64> {
+        if !self.initialized {
+            return None;
+        }
+        self.run_hook(HookType::Suitability)
+            .ok()
+            .as_ref()
+            .and_then(|hook_out| hook_out.stdout().lines().last())
+            .and_then(|line| line.trim().parse::<u64>().ok())
```

Comment:
Similar question here - if we can't parse the line into a `u64`, we get none, and so the suitability rules go back to default. This can be fine, but we loose any chance to warn the user that the hook isn't written right.